cdap-ui/app/cdap/components/ScrollableList/index.js

```diff
@@ -47,22 +47,32 @@ export default class ScrollableList extends Component {
     });
   }
   computeHeight() {
-    let targetDimensions = document.getElementById(this.props.target).getBoundingClientRect();
-    let bodyBottom = document.body.getBoundingClientRect().bottom;
-    let targetBottom = targetDimensions.bottom;
-    let heightOfList = bodyBottom - targetBottom;
-    let numberOfElemsInList = Math.floor(heightOfList / 39);
+    let heightOfList = document.body.getBoundingClientRect().height;
+    let numberOfElemsToDisplay = Math.floor(heightOfList / 29);
```

Comment:
Not sure where the number `29` coming from (is it the label for submenu?) but I am seeing this 
<img width="1355" alt="screen shot 2017-07-17 at 3 02 33 pm" src="https://user-images.githubusercontent.com/1452845/28291893-65e8fd04-6b01-11e7-965f-98d7d6618d1b.png">